spec/models/content_asset_spec.rb

```diff
@@ -1,5 +1,28 @@
 require 'rails_helper'
 
 RSpec.describe ContentAsset, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe "with an attached file" do
+    before(:each) do
+      @content_asset = FactoryBot.create(:content_asset)
+    end
+
+    it "has a file attached" do
+      expect(@content_asset.avatar).to be_attached
+    end
+  end
+
+  describe "required attributes" do
```

Comment:
These tests could also benefit from RSpecs support of one-liner syntax.

I was thinking something along these lines

```
describe "#valid" do
  subject { ContentAsset.new }

  it { should validate_presence_of(:title).with_message("can't be blank") }
  it { should validate_presence_of(:avatar).with_message("can't be blank") }
end
```

Reply:
this seems to require additional matchers to be setup